packages/core/src/__tests__/ceramic-pinning.test.ts

```diff
@@ -47,18 +47,22 @@ const createCeramic = async (
 
   const databaseFolder = await tmp.dir({ unsafeCleanup: true })
   const connectionString = new URL(`sqlite://${databaseFolder.path}/ceramic.sqlite`)
-  const ceramic = await Ceramic.create(ipfs, {
-    stateStoreDirectory,
-    anchorOnRequest,
-    indexing: {
-      db: connectionString.href,
-      allowQueriesBeforeHistoricalSync: true,
-      disableComposedb: true,
-      enableHistoricalSync: false,
+  const ceramic = await Ceramic.create(
+    ipfs,
+    {
+      stateStoreDirectory,
+      anchorOnRequest,
+      indexing: {
+        db: connectionString.href,
+        allowQueriesBeforeHistoricalSync: true,
+        disableComposedb: true,
+        enableHistoricalSync: false,
+      },
+      pubsubTopic: '/ceramic/inmemory/test', // necessary so Ceramic instances can talk to each other
```

Comment:
would like to see a constant for this

Reply:
this is just a test file, the value used here could be anything